doc/classes/ScrollContainer.xml

```diff
@@ -15,7 +15,12 @@
 			<return type="void" />
 			<argument index="0" name="control" type="Control" />
 			<description>
-				Ensures the given [code]control[/code] is visible (must be a direct or indirect child of the ScrollContainer). Used by [member follow_focus].
+				Ensures the given [code]control[/code] is visible (must be a direct or indirect child of the ScrollContainer). Used by [member follow_focus]. Note that this will not work on a node that was just added this frame. If you want to "autoscroll" to a newly added child, you must wait until the next frame like so:
+				[codeblock]
+				$VBoxContainer.add_child(child_node)
+				await get_tree().idle_frame
+				ensure_control_visible(child_node)
+				[/codeblock]
```

Comment:
```suggestion
				Ensures the given [code]control[/code] is visible (must be a direct or indirect child of the ScrollContainer). Used by [member follow_focus]. 
				[b]Note:[/b] This will not work on a node that was just added during the same frame. If you want to scroll to a newly added child, you must wait until the next frame using [signal SceneTree.process_frame]:
				[codeblock]
				add_child(child_node)
				await get_tree().process_frame
				ensure_control_visible(child_node)
				[/codeblock]
```